apps/storefront/src/pages/QuickOrder/components/QuickOrderFooter.tsx

```diff
@@ -233,20 +231,13 @@ function QuickOrderFooter(props: QuickOrderFooterProps) {
     setIsRequestLoading(true);
     handleClose();
 
-    const productIds = uniq(checkedArr.map(({ node }) => Number(node.productId)));
-
-    if (productIds.length === 0) {
-      snackbar.error(b3Lang('purchasedProducts.footer.selectOneItemToAdd'));
```

Comment:
This validation was deleted since we are already validating the length of checkedArr in the function handleOpenBtnList